src/lib/holocene/combobox/combobox.svelte

```diff
@@ -245,13 +244,13 @@
   </label>
 
   <div
-    class="text-sm w-full h-10 flex flex-row items-center rounded-lg bg-white border border-primary"
+    class="combobox-wrapper"
```

Comment:
Wondering if we want some padding on the left here when there isn't an icon

![Screenshot 2023-08-29 at 4 17 08 PM](https://github.com/temporalio/ui/assets/15069288/c481043a-b841-4078-92b7-619e4f896fa8)

Reply:
Yep good call, fixed that in one of my latest commits.